### PR TITLE
Add text and fix instances table on mobile devices

### PR DESCRIPTION
**Pull request by Gared**

### CODE REVIEW

File: src/pages/instances.tsx [hunk 1]
```diff
@@ -50,6 +50,12 @@ export const Instances = () => {
 
     return (
         <div className="m-5 flex flex-col h-screen"><h1 className="text-4xl font-bold mb-5">Scanned instances</h1>
+            <p>
+                This is an overview of all scanned Etherpad instances. You can start a new scan by entering the url of
```
Comment (SamTV12345) [inline src/pages/instances.tsx:54]: I think the URL should be capitalized ☺️

File: src/pages/instances.tsx [hunk 2]
```diff
@@ -50,6 +50,12 @@ export const Instances = () => {
...
+                the instance and clicking on the "Start scan" button.
```
Comment (SamTV12345) [inline src/pages/instances.tsx:55]: We could wrap Start Scan in a pre and set a different background like lightgrey.

**Review by SamTV12345 — APPROVED** [review]

File: src/pages/instances.tsx (hunk 1 above)
Comment (Gared) [reply src/pages/instances.tsx:54]: Done

File: src/pages/instances.tsx (hunk 2 above)
Comment (Gared) [reply src/pages/instances.tsx]: Done

**Comment by SamTV12345** [comment]
Awesome. Thanks Gared ☺️

**Pull request merged**